x-pack/plugins/maps/public/classes/layers/vector_layer/vector_layer.tsx

```diff
@@ -936,6 +939,10 @@ export class AbstractVectorLayer extends AbstractLayer implements IVectorLayer {
     return this.getSource().hasTooltipProperties() || this.getJoins().length > 0;
   }
 
+  isShowTooltip(): boolean {
```

Comment:
Existing `canShowTooltip` and `isShowTooltip` do not work well together and create a confusing API

I would recommend renaming showTooltips state to disableTooltips. Then rename isShowTooltip to areTooltipsDisabled. That should provide clearer API.

Would you mind adding comments to API for these methods that explain how they work together?